promscale/quick-start.md

```diff
@@ -0,0 +1,61 @@
+# Quick Start with Promscale
+Use Docker Compose to learn how Promscale is a single data store for your
+metrics and traces with full Prometheus and OpenTelemetry support, seamless
+integration with Grafana and Jaeger.  
+
+## Before you begin
+* Ensure that you have [installed Docker Compose][docker-compose].
+
+## Install Promscale with Docker Compose
+To install Promscale with Docker Compose clone the Promscale [repository][gh-promscale] and then run the Docker Compose. The `docker-compose` directory contains the services for the following components:
+* Promscale for analytics and long term storage of metrics and traces
+* Prometheus and Node exporter for metrics
+* OpenTelemetry collector, demo applications and Grafana for traces and visualisation
+
+### Installing Promscale with Docker Compose
+
+<procedure>
+
+1. Clone the repository that defines the services for Promscale:
+   ```bash
+   git clone git@github.com:timescale/promscale.git
+   ```
+1. Change to `promscale-demo` directory.
+   ```bash
+   cd docker-compose/promscale-demo
+   ```
+1. Run Promscale on Docker Compose:
+   ```bash
+   docker compose up -d
+   ```  
+
+</procedure >
+
+## Explore Promscale
+
+After Promscale is up and running you can explore the services installed in
+Docker Compose, such as Grafana. View the data sources that are configured to
+Promscale and the dashboards for trace data. Access Grafana on the browser
+`http://localhost:3000` as `admin` user using the password `admin`.
```

Comment:
```suggestion
When you have Promscale up and running, you can explore the services installed 
by Docker Compose. For example, you can use Grafana to see the data sources 
that are configured for Promscale, and use Grafana dashboards to visualize trace 
data. Access Grafana by navigating to `http://localhost:3000` in your web browser, 
and log in as `admin` with the password `admin`.
```

Reply:
This paragraph doesn't really match the heading. The heading is about exploring Promscale, while the para is all about Grafana. Why not list out a few different things here, rather than just the Grafana bits? Or, alternatively, change the heading to "Use Grafana with Promscale" or similar.